pkgs/cupertino_http/example/.gitignore

```diff
@@ -0,0 +1,47 @@
+# Miscellaneous
+*.class
+*.log
+*.pyc
+*.swp
+.DS_Store
+.atom/
+.buildlog/
+.history
+.svn/
```

Comment:
Is there some flutter template that sticks this junk in here?

If we need to keep this file, can we delete through line 23?